### ISSUE TITLE
Check UCSC datasets to avoid duplication of effort

### ISSUE THREAD

**Issue by ESapenaVentura** [ebi-ait/hca-ebi-wrangler-central#545]
**Description of the issue**
UCSC wranglers have given us a list of datasets they are working on:

- GSE76381
- GSE166175
- GSE165045
- GSE169396
- GSE176308
- GSE150212
- GSE174332
- GSE148907
- GSE108041
- GSE136314
- GSE102580
- GSE117498
- GSE129447
- phs000424
- GSE172176
- GSE185477
- GSE156753
- GSE184462
- GSE145154
- GSE179565
- GSE152290
- GSE166895
- GSE157329
- GSE163587

We need to check if we have them in ingest and don't do it unless already wrangled.

**Acceptance criteria**
- [x] All accessions have been checked in ingest
- [ ] (Datasets not in ingest) Add them to ingest
- [x] (Not-started datasets, in ingest) Add Rachel as primary wrangler in admin area
- [x] (Started/finished datasets, in ingest) Create a list and discuss

**Comment by ofanobilbao** [comment]
I have created a spreadsheet after checking all the accessions in Ingest. Sharing the spreadsheet here so that it can be shared with UCSC wranglers. It specifies if the project is or not in Ingest, and the state. For those projects that were already in Ingest I have assigned them to Rachel (except for one that was assigned to Paris already) and changed the status to Eligible, for UCSC Wranglers to change as they go. I believe that it would be easier if they create their own projects in Ingest rather than us doing that for them. @gabsie @ESapenaVentura what do you think? If that's agreed then we could share the doc with them and this ticket could be closed?

https://docs.google.com/spreadsheets/d/1kyVB6OqpzTEDInb6uXKjRmLRQRr9lYIt9L7PGLNHlU8/edit?usp=sharing

**Comment by ESapenaVentura** [comment]
@ofanobilbao That's exactly what I was thinking we could do. I have double checked and seems alright, it would be good to tell them to create the projects + assign a UCSC wrangler and an status to them!

**Comment by gabsie** [comment]
hey peeps I have created a meeting for us all for next wednesday to talk to them about all recent things ingest. if you guys wanted to do that sooner, while I am away, please feel free :). then next wednesday we can discuss generally other things, like the graph validator.

**Comment by ofanobilbao** [comment]
@gabsie meeting will need to be re-scheduled as Paris and Will can't make it

**Comment by ofanobilbao** [comment]
Enrique had a brief discussion with UCSC and they would appreciate if we could add them to Ingest this time

**Comment by ofanobilbao** [comment]
I feel like probably we should close this ticket. The only step missing is talk to UCSC on Friday. But we have already kind of hinted to it. Did we share the spreadsheet with them @ESapenaVentura ?

**Comment by ESapenaVentura** [comment]
Just to clarify, what are the outcomes of this ticket? Is it the idea that we'll speak with them on Friday and convince them to add those datasets when they can?

**Comment by ofanobilbao** [comment]
The ticket was mostly to avoid duplications. Not how we avoided them or how we created the missing projects in ingest. So I'd say, that sharing the SPS with UCSC should be enough to cover what this ticket was about. I think going forward the idea would be that they check ingest themselves not to duplicate. And that they create entries in Ingest when they are working on a project, right @gabsie ? But obviously this is all open to discussion with them. So I believe the ticket is done. Maybe we need to circulate the SPS with the rest of the wranglers in case anyone comes across any of the projects currently not in Ingest yet
